### PR TITLE
ui: various ui updates

### PR DESCRIPTION
**Pull request by czgu**
- Show tooltip about how to use data doc contents (table of contents) 
image

- Show Empty message if there are no scheduled docs
    - Also remove user select for empty message
- Make keyboard shortcuts slightly more readable

### CODE REVIEW

File: querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx [hunk 1]
```diff
@@ -60,7 +61,13 @@ export const DataDocLeftSidebar: React.FunctionComponent<IProps> = ({
                         icon="ArrowLeft"
                         onClick={() => setContentState('default')}
                     />
-                    <div>contents</div>
+                    <div className="flex-row">
+                        <span>contents</span>
```
Comment (meowcodes) [inline querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx:65]: does this need any margin? can you screenshot

File: querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx [hunk 2]
```diff
@@ -125,6 +126,9 @@ const DataDocScheduleList: React.FC = () => {
                         key={docWithSchedule.doc.id}
                     />
                 ))}
+                {dataDocsWithSchedule.length === 0 && (
+                    <EmptyText>No DataDocs found</EmptyText>
```
Comment (meowcodes) [inline querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx:130]: No Scheduled DataDocs

Comment (czgu) [reply querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx:130]: Docs doesn't have to be scheduled to be in this list

File: querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx [hunk 3]
```diff
@@ -60,7 +61,13 @@ export const DataDocLeftSidebar: React.FunctionComponent<IProps> = ({
...
+                        <InfoButton layout={['right', 'top']}>
+                            Click to jump to the corresponding cell. Drag the
+                            cells to reorder them.
```
Comment (meowcodes) [inline querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx:68]: Drag cells to reorder them. (No the)

**Pull request closed without merging**